uint16 finger0
uint16 finger1
uint16 finger2
uint16 finger3
